include/fiction/algorithms/physical_design/graph_oriented_layout_design.hpp

```diff
@@ -1899,28 +1901,15 @@ class graph_oriented_layout_design_impl
      */
     void initialize_pis_cost_and_num_expansions() noexcept
     {
-        static constexpr std::array<pi_locations, 60> pi_locs = {
-            {pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT,
-             pi_locations::TOP, pi_locations::LEFT, pi_locations::TOP_AND_LEFT}};
-
-        auto pi_loc_it = pi_locs.cbegin();
+        static constexpr std::array<pi_locations, 3> pattern{pi_locations::TOP, pi_locations::LEFT,
+                                                             pi_locations::TOP_AND_LEFT};
```

Comment:
The following syntax might work:

```suggestion
        static constexpr std::array pattern{pi_locations::TOP, pi_locations::LEFT,
                                                             pi_locations::TOP_AND_LEFT};
```